client/components/sites-dropdown/index.jsx

```diff
@@ -30,7 +30,8 @@ export default React.createClass( {
 		indicator: React.PropTypes.bool,
 		autoFocus: React.PropTypes.bool,
 		onClose: React.PropTypes.func,
-		onSiteSelect: React.PropTypes.func
+		onSiteSelect: React.PropTypes.func,
+		filter: React.PropTypes.func
```

Comment:
Yeah, I was gonna mention the same thing. Good opportunity to document all the props.